<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.8.11">
  <compounddef id="classfranka__tut__reasoning_1_1errors_1_1Error" kind="class" language="Python" prot="public">
    <compoundname>franka_tut_reasoning::errors::Error</compoundname>
    <basecompoundref prot="public" virt="non-virtual">Exception</basecompoundref>
    <derivedcompoundref refid="classfranka__tut__reasoning_1_1errors_1_1Grasping__Error" prot="public" virt="non-virtual">franka_tut_reasoning.errors.Grasping_Error</derivedcompoundref>
    <derivedcompoundref refid="classfranka__tut__reasoning_1_1errors_1_1LocationUnknown__Error" prot="public" virt="non-virtual">franka_tut_reasoning.errors.LocationUnknown_Error</derivedcompoundref>
    <derivedcompoundref refid="classfranka__tut__reasoning_1_1errors_1_1RequestSemantic__Error" prot="public" virt="non-virtual">franka_tut_reasoning.errors.RequestSemantic_Error</derivedcompoundref>
    <derivedcompoundref refid="classfranka__tut__reasoning_1_1errors_1_1RequestSyntax__Error" prot="public" virt="non-virtual">franka_tut_reasoning.errors.RequestSyntax_Error</derivedcompoundref>
    <derivedcompoundref refid="classfranka__tut__reasoning_1_1errors_1_1TargetUnknown__Error" prot="public" virt="non-virtual">franka_tut_reasoning.errors.TargetUnknown_Error</derivedcompoundref>
    <derivedcompoundref refid="classfranka__tut__reasoning_1_1errors_1_1WidthUnknown__Error" prot="public" virt="non-virtual">franka_tut_reasoning.errors.WidthUnknown_Error</derivedcompoundref>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="classfranka__tut__reasoning_1_1errors_1_1Error_1a7b2da2792da7abb0417f7d3128b8811f" prot="public" static="no" const="no" explicit="no" inline="no" virt="non-virtual">
        <type>def</type>
        <definition>def franka_tut_reasoning.errors.Error.__init__</definition>
        <argsstring>(self)</argsstring>
        <name>__init__</name>
        <param>
          <type>self</type>
          <defname>self</defname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="franka_tut_reasoning/src/franka_tut_reasoning/errors.py" line="12" column="1" bodyfile="franka_tut_reasoning/src/franka_tut_reasoning/errors.py" bodystart="12" bodyend="14"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
<para><verbatim>Base class for exceptions in this module.</verbatim> </para>    </detaileddescription>
    <inheritancegraph>
      <node id="123">
        <label>franka_tut_reasoning.errors.WidthUnknown_Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1WidthUnknown__Error"/>
        <childnode refid="116" relation="public-inheritance">
        </childnode>
      </node>
      <node id="117">
        <label>Exception</label>
      </node>
      <node id="119">
        <label>franka_tut_reasoning.errors.LocationUnknown_Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1LocationUnknown__Error"/>
        <childnode refid="116" relation="public-inheritance">
        </childnode>
      </node>
      <node id="122">
        <label>franka_tut_reasoning.errors.TargetUnknown_Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1TargetUnknown__Error"/>
        <childnode refid="116" relation="public-inheritance">
        </childnode>
      </node>
      <node id="120">
        <label>franka_tut_reasoning.errors.RequestSemantic_Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1RequestSemantic__Error"/>
        <childnode refid="116" relation="public-inheritance">
        </childnode>
      </node>
      <node id="121">
        <label>franka_tut_reasoning.errors.RequestSyntax_Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1RequestSyntax__Error"/>
        <childnode refid="116" relation="public-inheritance">
        </childnode>
      </node>
      <node id="118">
        <label>franka_tut_reasoning.errors.Grasping_Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1Grasping__Error"/>
        <childnode refid="116" relation="public-inheritance">
        </childnode>
      </node>
      <node id="116">
        <label>franka_tut_reasoning.errors.Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1Error"/>
        <childnode refid="117" relation="public-inheritance">
        </childnode>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="125">
        <label>Exception</label>
      </node>
      <node id="124">
        <label>franka_tut_reasoning.errors.Error</label>
        <link refid="classfranka__tut__reasoning_1_1errors_1_1Error"/>
        <childnode refid="125" relation="public-inheritance">
        </childnode>
      </node>
    </collaborationgraph>
    <location file="franka_tut_reasoning/src/franka_tut_reasoning/errors.py" line="10" column="1" bodyfile="franka_tut_reasoning/src/franka_tut_reasoning/errors.py" bodystart="10" bodyend="15"/>
    <listofallmembers>
      <member refid="classfranka__tut__reasoning_1_1errors_1_1Error_1a7b2da2792da7abb0417f7d3128b8811f" prot="public" virt="non-virtual"><scope>franka_tut_reasoning::errors::Error</scope><name>__init__</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
